presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaInternalFieldDescription.java

```diff
@@ -42,6 +42,11 @@
      */
     PARTITION_ID_FIELD("_partition_id", BigintType.BIGINT, "Partition Id"),
 
+    /**
+     * <tt>_timestamp</tt> - Kafka message timestamp.
+     */
+    TIMESTAMP_FIELD("_timestamp", BigintType.BIGINT, "Message creation timestamp"),
```

Comment:
As far as I know, the timestamp definition can be changed by `message.timestamp.type` property (CreateTime or LogAppendTime). Also, it always returns `-1` if magic is 0.

Reply:
Maybe we should have columns `_created` and `_appended` so it would pass the semantic in the name not the date type. Also I would use `TimestampWithTimeZoneType#TIMESTAMP_WITH_TIME_ZONE` data type instead of `BIGINT`.